### PR TITLE
chore: upgrade to v3

### PR DESCRIPTION
**Pull request by matthew1809**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,25 +9,25 @@
...
     "fs": "^0.0.1-security",
     "lokijs": "^1.5.12",
-    "next": "13.4.4",
+    "next": "^14.2.11",
     "pino-pretty": "^10.0.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "tls": "^0.0.1",
     "typescript": "5.1.3",
-    "viem": "^1.1.7",
```
Comment (guiMendel) [inline package.json:30]: why are we removing viem from the deps?

**Review by guiMendel — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (matthew1809) [reply package.json:30]: Don't we install it for you @guiMendel ?

Comment (guiMendel) [reply package.json:30]: only when using `npm i`, it's generally better to leave it explicitly

Comment (matthew1809) [reply package.json:30]: ok fair, fixed

**Pull request merged**